src/main/java/controllers/TableViewController.java

```diff
@@ -0,0 +1,136 @@
+package controllers;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.beans.value.ObservableValue;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableColumn.CellDataFeatures;
+import javafx.scene.control.TableView;
+import javafx.util.Callback;
+import model.data.DataColumn;
+import model.data.DataModel;
+import model.data.DataRow;
+import model.data.DataTable;
+import model.data.Row;
+import model.data.value.StringValue;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Observable;
+import java.util.Observer;
+import java.util.logging.Logger;
+
+/**
+ * Controls the visualization of the table in the user interface.
+ * @author Paul
+ *
+ */
+public class TableViewController implements Observer {
+	
+	private Logger logger = Logger.getLogger("TabViewController");
+	
+	@FXML 
+	private TableView<ObservableList<StringProperty>> tableView;
+	@FXML
+	private ListView<DataTable> inputTables;
+	
+	private DataTable currentTable;
+	private DataModel model;
+
+	
+	/**
+	 * Creates a new TableViewController.
+	 */
+	public TableViewController() {
+	}
+	
+	/**
+	 * Loads the data from the model and updates the view for the user.
+	 */
+	private void fillTable() {
+		logger.info("update table");
```

Comment:
Misschien handig om als je toch logt de naam van de tabel er in ieder geval in te zetten?